django/db/backends/mysql/validation.py

```diff
@@ -3,6 +3,23 @@
 
 
 class DatabaseValidation(BaseDatabaseValidation):
+    def check(self, **kwargs):
+        warnings = super(DatabaseValidation, self).check(**kwargs)
```

Comment:
"warnings" -> "issues" for consistency with elsewhere?
I would also create a separate "check_sql_mode()" method so if more checks are added, we don't end up with a giant method (similar to the pattern in `Field.check()`).